gapis/api/vulkan/api/queued_command_tracking.api

```diff
@@ -267,7 +267,7 @@ sub void callCommand(ref!CommandReference reference) {
     case cmd_vkCmdBeginRenderPass:
       dovkCmdBeginRenderPassCommon(CommandBuffers[reference.Buffer].BufferCommands.vkCmdBeginRenderPass[reference.MapIndex])
     case cmd_vkCmdNextSubpass:
-      dovkCmdNextSubpass(CommandBuffers[reference.Buffer].BufferCommands.vkCmdNextSubpass[reference.MapIndex])
```

Comment:
Should we be checking these keys are present in the map and reporting errors where appropriate?

Reply:
They should exist. If they do not exist, there is nothing we cannot do anything but panic. Because no user application can trigger that case.

And having a panic or a crash does not matter at that point. It might be cleaner to do the panic but again, there is an existing pattern and if we want to do that, we need to do it for the entire file and not part of this PR.